### ISSUE TITLE
Improve colour contrast of Crown Copyright logo

### ISSUE THREAD

**Issue by CharlotteDowns** [alphagov/govuk-frontend#2134]
## What
The crown copyright logo against the footer background has a low contrast ratio. However, it does meet the contrast requirement for [meaningful graphics](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) of 3:1. We don't think this is a WCAG fail and it doesn't need to be listed in an accessibility statement, but there's potential work we can do to improve the contrast so it better matches that of the text/links around it.

![current Crown copyright logo in the footer](https://user-images.githubusercontent.com/56260216/106613192-d7b30980-6561-11eb-8f5b-9ffa8df5c47d.jpg)
The crown copyright logo has a hex code colour of `#6E777A`, which against the footer background of `#F3F2F1` has a AA Large 4.09:1 colour contrast ratio.

Normal Text
WCAG AA:Fail
WCAG AAA:Fail

Large Text
**WCAG AA:Pass**
WCAG AAA:Fail

Graphical Objects and User Interface Components
**WCAG AA:  Pass**

## Why
[A comment was added to the epic about an issue raised in the Axe Review accessibility tool](https://github.com/alphagov/govuk-frontend/issues/1680#issuecomment-761044742). The failure is only of the logo and not to the footer text.

## Who needs to know about this
Designers, Developers, Policy

## Further detail
A suggestion could be to change the colour of the logo but we would need to check with the owners of the Crown Copyright logo.

![Crown copyright logo in the footer using the darker text colour](https://user-images.githubusercontent.com/56260216/106612745-578ca400-6561-11eb-9b1e-a2255c12ea56.jpg)
Here is a mock-up of the crown copyright logo using the footer text colour `#454A4C` which has a contrast ratio of AAA 8.03:1 against the footer background `#F3F2F1`.

## Done when

- [ ] Agreed with Crown Copyright on approach
- [ ] Decided on whether the logo should be an .svg or .png
- [ ] Updated file created
- [ ] Pull request created and merged

**Comment by vanitabarrett** [comment]
Updated issue to explain that the Crown Copyright logo meets colour contrast requirements for non-text elements and meaningful graphics, so this isn't a WCAG fail.

**Comment by davidc-gds** [comment]
If we want to exactly pass 4.5:1 contrast ratio without rocking the boat too much, we could go with #667171.

That would be the closest colour to what we currently have, so would likely be barely noticeable by the general public (but will mean we meet the 4.5:1 threshold).

#6E777A Current colour at 4.09:1
#667171 Proposed colour at 4.5:1
